### ISSUE TITLE
assertion failed: s->desiredAvailable <= s->leftUntilDone

### ISSUE THREAD

**Issue by jorti** [transmission/transmission#4999]
### What is the issue?

Transmission-daemon 4.0.0 crashes with this message:

~~~
assertion failed: s->desiredAvailable <= s->leftUntilDone (/builddir/build/BUILD/transmission-4.0.0/libtransmission/torrent.cc:1638)
Aborted (core dumped)
~~~

### Which application of Transmission?

transmission-daemon

### Which version of Transmission?

transmission-daemon 4.0.0 (280ace12f8)

**Comment by ckerr** [comment]
Hi @jorti, thanks for reporting this.

This bug is fixed by #4879. The fix will be in `4.0.1`, which should be released shortly.

**Comment by ckerr** [comment]
Hi @jorti, thanks for reporting this.

This looks like a duplicate of https://github.com/transmission/transmission/issues/4695, so I'm closing this ticket as a dupe.

The fix for this will be included in `4.0.1`, which should be released shortly.
